src/test/suite/registryWorkspace.test.ts

```diff
@@ -44,17 +44,18 @@ suite('Tutorial Registry Test Suite', () => {
 		await cleanFiles();
 	});
 
-	test('create new didact file and register it as a tutorial in the view', async () => {
-		const didactUriToCreateFile = `didact://?commandId=vscode.didact.scaffoldProject&extFilePath=redhat.vscode-didact/examples/register-tutorial.project.json`;
+	test('copy didact file into workspace and register it as a tutorial in the view', async () => {
```

Comment:
This change was not precipitated by the PR, but the test failure started cropping here. Went ahead and fixed it.

Reply:
woudl be better to provide a seprate PR. Especially given that you are merging all commits and we cannot review the commit history.